appdaemon/config.yaml

```diff
@@ -27,12 +27,14 @@ map:
 options:
   system_packages: []
   python_packages: []
+  cache_packages: false
```

Comment:
I'm not understanding why this needs to be an option?

Are there drawbacks?

Reply:
Not really, just didn't want to force it to everyone and I can't be 100% sure that it wont break someone's installation if they're using `python_packages` in a weird way. I'd be happy to remove it.

Reply:
In that case, remove it.

However, this will not resolves system packages.

Reply:
Not sure I understood what you mean by "this will not resolves system packages". Right now `cache_packages` controls caching for both system and python packages. If true both are cached, if false both have the previous behaviour.

You want to remove it, and always cache both system and python packages, right?